data/human/human missions.txt

```diff
@@ -7454,3 +7454,599 @@ mission "Adelita 5"
+			choice
+				`	"Sounds like a good plan to me."`
+					goto choseplana
+				`	"What would it take to boost profits even more?"`
+					goto planb
+				`	"And if I have concerns about the environmental impact?"`
+					goto planc
+			label planb
+			`	"We could tap into some unregulated fuel sources. No oversight, no regulations. The fuel's cheaper and abundant, but it's volatile--one wrong move, and it's a disaster waiting to happen. The profits, though, are through the roof. For the possible cost of a few lives and some bad PR, we can boost returns to around 4100 credits a day, or about 15% a year."`
+			choice
+				`	"Bring on the profits, this unregulated option is the one for me."`
+					goto choseplanb
+				`	"Tell me about the conventional expansion again."`
+					goto plana
+				`	"And if I have concerns about the environmental impact?"`
+					goto planc
+			label planc
+			`	"There's also the eco-friendly route. We'd invest in green energy technologies and refining processes that minimize environmental impact and implement best-in-class workplace safety protocols. It's the most expensive option, and it'll slow our expansion, but it would reduce our environmental liabilities and might give us some goodwill. Easily the worst returns with around 2000 credits a day expected, or just around 7% a year."`
+			choice
+				`	"This environmental and safety focused option is the one for me."`
+					goto choseplanc
+				`	"Tell me about the conventional expansion again."`
+					goto plana
+				`	"What would it take to boost profits even more?"`
+					goto planb
+			label choseplana
+			action
+				"salary: Syndicate Business" += 2700
+				set "Syndicate Business expansion 10m neutral"
+			`	"Excellent choice. The conventional expansion is a safe bet with solid returns. It's not the flashiest option, but the steady stream of ships passing through Crossroads will ensure consistent profits. The only drawback is that it won't make headlines--but it'll keep the credits flowing smoothly."`
+				goto ending
+			label choseplanb
+			action
+				"salary: Syndicate Business" += 4100
+				set "Syndicate Business expansion 10m evil"
+			`	"Well done. The unregulated option is bold and ambitious--exactly the kind of move the Syndicate prizes. The volatility of this fuel isn't for the faint of heart, but some bad PR is well worth the profits and potential learnings we'll achieve. High risk often means high reward."`
+				goto ending
+			label choseplanc
+			action
+				"salary: Syndicate Business" += 2000
+				set "Syndicate Business expansion 10m good"
+			`	"An admirable decision. The eco-friendly route won't make the most credits, but it's a forward-thinking investment. It could pay off in other ways, like reduced turnover and environmental liabilities. Objectively the worst investment from a returns perspective, but I suppose you value other things more."`
+			label ending
```

Comment:
```suggestion
			label ending
			action
				log "People" "Howard Vendeur" "Apparently Howard has the authority to completely change which way investments with his firm go."
```

Reply:
What I meant by that was he can agree to what one investor says without consulting others.
But this is another unnecessary one.